application/views/pc/pages/sunglasses/imagesByModelFlexSlider.php

```diff
@@ -57,6 +57,15 @@ class="width-75-percent"
                                 <?php endif; ?>
 								 </p>
 							</div> <!-- end .container -->
+							<div id="infograph">
+							  <div class="center-infograph">
```

Comment:
There's no such word as infograph :)
Let's call it `shop-benefits-infographic__inner-container`? And do we need this div at all?